### ISSUE TITLE
expand "how to hire appsec"

### ISSUE THREAD

**Issue by DinisCruz** [DinisCruz/Book_SecDevOps_Risk_Workflow#12]
"ability to write Tests in TTD (with CI workflow) is more important than the language"

see thread https://twitter.com/zeroXten/status/786356626060087296

**Comment by davevs** [comment]
On a higher level you can even argue that being able to define evil user stories and undesired functional test cases is what it's all about. The actual creation can be left to test automation experts, possibly creating a champion in the process.

**Comment by DinisCruz** [comment]
yes but I find that being able to write tests is quite important since it will expose that person to a lot of the dev workflows 

related to the idea [Using tests to communicate](https://github.com/DinisCruz/Book_SecDevOps_Risk_Workflow/blob/bbb2f5ee5c3e8a0e1f9cca43b47019e8fcfc1d25/content/2.Risk-workflow/For-management/Using-tests-to-communicate.md)
